clang/lib/CIR/Dialect/IR/CIRTypes.cpp

```diff
@@ -408,20 +412,23 @@ llvm::TypeSize
 DataMemberType::getTypeSizeInBits(const ::mlir::DataLayout &dataLayout,
                                   ::mlir::DataLayoutEntryListRef params) const {
   // FIXME: consider size differences under different ABIs
+  assert(!CIRDialectUnimplementedFeature::cxxABI());
```

Comment:
Using the name suggested in the next comment, this becomes `assert(!cir::MissingFeature::cxxABI())`.

Reply:
Updated.